src/plugins/discover/public/services/doc_views/components/doc_viewer_source/source.test.tsx

```diff
@@ -17,11 +17,11 @@ import { JsonCodeEditorCommon } from '../../../../components/json_code_editor/js
 import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
 import { buildDataTableRecord } from '../../../../utils/build_data_record';
 
-const mockIndexPattern = {
+const mochDataView = {
```

Comment:
```suggestion
const mockDataView = {
```

Surprising with how many changes, this is the first typo I saw 😁

Reply:
Oh looks like you got to this before I submitted my review and this is fixed.

Reply:
Yeah, I was also surprise because it's usually a thing my brain would tell me: it's ok 😄

Reply:
Anyway catching this typo in 167 files it deserves a title, what do you prefer: 
* Eagle Eye Davis
* Needle Haystack Davis
?

Reply:
"Eagle Eye Davis" has a ring to it!